presto-main/src/main/java/com/facebook/presto/operator/OperatorContext.java

```diff
@@ -83,6 +84,8 @@
     private final AtomicLong finishCpuNanos = new AtomicLong();
     private final AtomicLong finishUserNanos = new AtomicLong();
 
+    private final CounterStat hashCollisions = new CounterStat();
```

Comment:
Why do you need `CounterStat` for this? Simple `AtomicLong` or double isn't enough?